core/domain/learner_progress_services.py

```diff
@@ -616,3 +735,142 @@ def get_collection_summary_dicts(collection_summaries):
         })
 
     return summary_dicts
+
+
+def get_activity_progress(user_id):
+    """Returns the progress of the learners - the explorations and collections
+    completed by the user and those in progress.
+
+    Args:
+        user_id: str. The id of the learner.
+
+    Returns:
+        LearnerProgress. The learner progress domain object corresponding to the
+            particular learner.
+        dict. The number of entities that are no longer present.
```

Comment:
number --> numbers

Though really you should enumerate the keys of the dict, and the types/meaning of their corresponding values.